glossary.asciidoc

```diff
@@ -16,13 +16,13 @@ Comment from Gitter:
 ////
 
 Account::
-    An object containing an address, balance, and nonce, and optional storage and code. An account can be a contract account or an EOA (externally owned account).
+    An object containing an address, balance, nonce, and optional storage and code. An account can be a contract account or an EOA (externally owned account).
 
 Address::
-    Most generally, this represents an EOA or contract, which can receive (destination address) or send (source address) transactions on the blockchain. More specifically, it is the right-most 160 bits of a Keccak hash of an ECDSA public key.
+    Most generally, this represents an EOA or contract thath can receive (destination address) or send (source address) transactions on the blockchain. More specifically, it is the right-most 160 bits of a Keccak hash of an ECDSA public key.
```

Comment:
thath -> that?

Reply:
'that'

How embarrassing! I can't believe I missed my own typo. :(

Sorry about that, Javier.

--Chris

Reply:
To err is human. :-)